.github/workflows/josephdelacruzzz.yml

```diff
@@ -0,0 +1,21 @@
+name: Test for josephdelacruzz
+on: push
+jobs: 
+  my-first-job:
+    name: Test job for josephdelacruzzz
+    runs-on: ubuntu-latest
+    steps:
+      -name: Checkout repository
```

Comment:
Your spacing is slightly off. There should be a space after the `-`, like `- name:`. Your `uses`, `with` and `run` should also have one more space.

I found this error by checking [your Workflow run](https://github.com/josephdelacruzzz/git-tutorial-lab/actions/runs/11205774815) in the "Actions" tab of your GitHub repository, where your run was:
![image](https://github.com/user-attachments/assets/aa44f16c-fa5f-41bf-87c5-c147818b23be)